services/QuillLMS/engines/evidence/spec/lib/check_spec.rb

```diff
@@ -191,5 +191,64 @@ module Evidence
         end.to raise_error(Check::NoMatchedFeedbackTypesError)
       end
     end
+
+    context 'normalize_entry_text' do
```

Comment:
It would be nice to see an additional test for a string with all of these `utf-8` characters and not the codes. That would be a nice full process test that would also verify that the codes are the correct ones for the characters you expect.

Reply:
That seems reasonable.  I've added a case that explicitly checks that normalizing a string of all of the intended characters works.